### PR TITLE
add disabled_at and disable_feedback

### PR DESCRIPTION
**Pull request by Bee-lee**
# Description
I'm having problems with my local ingress so I want to see this in QA. 
The format is the same as our other timestamps, so the frontend should be expecting it -- also wanna see this on UI.

CCXDEV-5290

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- 
## Testing steps
`make before_commit`

## Checklist
* [x] `make before_commit` passes
* [x] updated documentation wherever necessary
* [x] added or modified tests if necessary
* [x] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by tisnik — APPROVED** [review]
That one is short one :)

File: server/rules.go [hunk 1]
```diff
@@ -160,6 +160,7 @@ func (server HTTPServer) getFeedbackAndTogglesOnRule(
 		rule.Disabled = false
 	} else {
```
Comment (dpensi) [inline server/rules.go:161]: wrong indentation?

File: server/rules.go [hunk 2]
```diff
@@ -168,6 +169,7 @@ func (server HTTPServer) getFeedbackAndTogglesOnRule(
 		rule.UserVote = types.UserVoteNone
```
Comment (dpensi) [inline server/rules.go:169]: same here, wrong indentation?

**Review by dpensi — APPROVED** [review]

File: server/rules.go (hunk 2 above)
Comment (Bee-lee) [reply server/rules.go:169]: it looks ok in my ide, probably just line breaks on github?

**Review by joselsegura — APPROVED** [review]

**Pull request merged**